designsafe/apps/workspace/admin.py

```diff
@@ -48,10 +61,10 @@ def get_fieldsets(self, request, obj=None):
                 },
             ),
             (
-                "HTML App Body for app_type: HTML",
+                "HTML or External app information",
                 {
                     "classes": ["collapse"],
-                    "fields": ["html"],
+                    "fields": ["external_href", "html",],
```

Comment:
**Idea.** This new feature could deprecate `html` property if all apps choose to link directly to app. C.J. (ux designer) and S.M. (DS app user & guide author & contributor) proposed this before launch. I was too afraid to change models before launch.

Reply:
I think that would need to be run by Tracy and Ellen. I also hesitate to deprecate it completely in the case an "HTML" app is wanted in the workspace browser

Reply:
Understood. I would run it by them.

This new `external_href` feature, as is, can be used to bypass workspace for direct downloads **only as requested** like in [<del>WA-326</del> <ins>WP-664</ins>](https://tacc-main.atlassian.net/browse/WP-664).

Reply:
Idea of `html` deprecation aside, what concerns do you have with `external_href` or my implementation of it?